google/cloud/forseti/common/email_templates/inventory_summary.jinja

```diff
@@ -17,27 +17,19 @@ limitations under the License.
 <!doctype html>
 <html>
 <body>
-    Timestamp: {{ snapshot_timestamp }}; {{ snapshot_time }}<br>
-    Status Summary: {{ status_summary }}<br>
+    inventory_index_id: {{ inventory_index_id }}<br>
```

Comment:
Can we add a timestamp here so the user knows when the inventory process is started/completed?